config_template.cfg

```diff
@@ -770,7 +770,17 @@ GILES_EXTRA_RELAXFACTOR= ( 0.05, 0.05)
 %
 % YES Non reflectivity activated, NO the Giles BC behaves as a normal 1D characteristic-based BC
 SPATIAL_FOURIER= NO
+% ------------------------ WALL ROUGHNESS DEFINITION --------------------------%
+% Type of wall (SMOOTH or ROUGH). By default all walls are assumed to be smooth.
+% If multiple walls are present, list type for each of them (ex- SMOOTH, ROUGH,..) 
+% in the order they are listed in MARKER_HEATFLUX definition. If all walls are smooth
+% this option can be removed.
+%WALL_TYPE= SMOOTH
```

Comment:
You have to list here all markers specified in MARKER_HEATFLUX **and** MARKER_ISOTHERMAL, right ?

Reply:
Yes, you are right. Will fix this and the other comments soon. 
Thanks.